todolist.js

```diff
@@ -0,0 +1,16 @@
+document.addEventListener("DOMContentLoaded", () => {
+let newUl = document.querySelector("ul")
+let button = document.querySelector(".sub")
+let submit = document.querySelector(".int")
+let res = document.querySelector(".reset")
+button.addEventListener("click", (event)=> {
+
+  let newInput = document.createElement("li")
+  newInput.innerText = submit.value
+  newUl.appendChild(newInput)
+  event.preventDefault();
+  res.reset();
+})
```

Comment:
Great work. It is best practice to place the preventDefault at the top of the function call, with the rest of your logic below (put on line 7 instead of line 11).